source/model/matchStatus.mc

```diff
@@ -14,11 +14,13 @@ class MatchStatus {
     private var p1TieBreakScore;
     private var p2TieBreakScore;
 
+    private var TotalDeuces;
+
     static function New() {
-        return new MatchStatus(0,0,0,0,0,0,0,0);
+        return new MatchStatus(0,0,0,0,0,0,0,0,0);
     }
 
-    function initialize(p1Sets, p2Sets, p1Games, p2Games, p1ScoreIdx, p2ScoreIdx, p1TieBreakScore, p2TieBreakScore) {
+    function initialize(p1Sets, p2Sets, p1Games, p2Games, p1ScoreIdx, p2ScoreIdx, p1TieBreakScore, p2TieBreakScore, TotalDeuces) {
```

Comment:
same, lets write var in camelCase instead of PascalCase

Reply:
Yes, I had noticed that all the variables had a pattern, then I got confused and did as I usually do, not that I follow a precise pattern like yours, but you know everyone has their own habits :D

Reply:
tbh i'm not 100% sure, but it seems that monkey C standard is to write like this. on the other hand, when bootstraping the app with the template from the SDK, no rule seems to be followed 😅  at least we try to be consistent 👍